### PR TITLE
FORMS-3589 Enable responsiveness for forms guide container component.

### PR DESCRIPTION
**Pull request by sakshi-arora1**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/container.html [hunk 1]
```diff
@@ -22,15 +22,15 @@
       data-cmp-is="adaptiveFormContainer"
       data-cmp-context-path="${request.contextPath}"
       data-cmp-path="${resource.path}"
-      class="cmp-adaptiveform-container ${grid.cssClass} cmp-container${wcmmode.edit ? ' {0}': '' @ format=[allowed.cssClass]}">
```
Comment (sakshi-arora1) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/container.html:25]: @rismehta Why were we doing this?
`cmp-container${wcmmode.edit ? ' {0}': '' @ format=[allowed.cssClass]}`

Comment (rismehta) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/container.html:25]: We don't need the CSS class in edit mode

**Comment by rismehta** [comment]
Can you also check the flaky test which was found in this run, https://app.circleci.com/pipelines/github/adobe/aem-core-forms-components/2346/workflows/fe2558e6-db87-4258-afe2-b3d9426c9cdd/jobs/7153

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/container.html (hunk 1 above)
Comment (sakshi-arora1) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/container.html:25]: well we don't need this at all.

**Comment by sakshi-arora1** [comment]
> Can you also check the flaky test which was found in this run, https://app.circleci.com/pipelines/github/adobe/aem-core-forms-components/2346/workflows/fe2558e6-db87-4258-afe2-b3d9426c9cdd/jobs/7153

This is already marked flaky with 3 retries.

File: ui.tests/test-module/specs/wizard/wizard.authoring.spec.js [hunk 2]
```diff
@@ -87,23 +87,23 @@ describe('Page - Authoring', function () {
             cy.deleteComponentByPath(wizardLayoutDrop) ;
         });
 
-        // todo: flaky
-        it('verify Navigation Working between tabs in Authoring', { retries: 3 }, function(){
+        it('verify Navigation Working between tabs in Authoring', function(){
```
Comment (sakshi-arora1) [inline ui.tests/test-module/specs/wizard/wizard.authoring.spec.js:90]: @rismehta / @deepprakash345 I have removed retries and moved the logic to execute post wait.

**Pull request merged**